cartographer/mapping/internal/3d/local_trajectory_builder_3d.cc

```diff
@@ -177,12 +177,11 @@ LocalTrajectoryBuilder3D::AddRangeData(
     const auto voxel_filter_duration = voxel_filter_stop - voxel_filter_start;
 
     if (sensor_duration.has_value()) {
-      double voxel_filter_fraction =
+      const double voxel_filter_fraction =
           std::chrono::duration_cast<std::chrono::duration<double>>(
```

Comment:
Use `common::ToSeconds()` which was added in #1244? Similar below.

Reply:
ok, I'll do that once #1244 is merged.